Gems/ROS2/Code/Source/Communication/QoS.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ROS2/Communication/QoS.h"
```

Comment:
repeat: angle brackets should be used for this include.
The only time where double quotes are used is when the header file being included is within the same directory.